src/ui/KonvergoWindow.cpp

```diff
@@ -106,7 +106,7 @@ KonvergoWindow::KonvergoWindow(QWindow* parent) :
 #endif
 
 #ifdef KONVERGO_OPENELEC
-  setVisibility(QWindow::FullScreen);
+  setWindowFullScreen(true);
 #else
```

Comment:
For the window method, `setFullScreen` name should be preferred.

Reply:
`setFullScreen` is already in use by another method, though, and it serves a different purpose of updating the application setting. What name should I use in light of that?

Reply:
Good point, i guess that name is fine then, at least i have no better idea.